src/serverenvironment.cpp

```diff
@@ -1670,6 +1670,39 @@ void ServerEnvironment::deleteParticleSpawner(u32 id, bool remove_from_object)
 	}
 }
 
+u32 ServerEnvironment::addNew3DLine()
+{
+	u32 id = 0;
```

Comment:
To avoid ID conflicts of deleted/added lines (may it be due to server lag or delayed mod calls) the most recent ID should be cached to continue counting from there. See `ActiveObjectMgr::getFreeId` for reference.